projects/slap_manipulation/src/slap_manipulation/policy/components.py

```diff
@@ -3,6 +3,26 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+# Copyright (c) 2023 PyG Team <team@pyg.org>
```

Comment:
this probably should not be here? presumably only applies to SOME of these components right?

Reply:
I'll link this license to specific methods then. Legal encouraged me to keep all license and wording information from every source we include directly in our code-base.